<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaQtAsignatura</class>
 <widget class="QTabWidget" name="VistaQtAsignatura">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vista de una asignatura</string>
  </property>
  <property name="movable">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="tabProfesores">
   <attribute name="title">
    <string>Profesores</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAnadirProfesor">
        <property name="text">
         <string>Añadir profesor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonQuitarProfesor">
        <property name="text">
         <string>Quitar profesor</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetProfesores">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>103</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nombre</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Correo electrónico</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tabCompaneros">
   <attribute name="title">
    <string>Compañeros</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="tabSesiones">
   <attribute name="title">
    <string>Sesiones</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="tabCitas">
   <attribute name="title">
    <string>Citas</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QWidget" name="tabNotas">
   <attribute name="title">
    <string>Notas</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
